### ISSUE TITLE
Make a constant for the precision of reading.

### ISSUE THREAD

**Issue by jmfloyd** [barbudor/CircuitPython_INA3221#2]

**Comment by jmfloyd** [comment]
(I hit return accidentally and it sent off the title!)
Suggest
Define constants for
MIN_SHUNT_VOLTAGE = const(0.00004)
MIN_BUS_VOLTAGE = const(0.008)

and use in code

This allows  calling methods to compare values against the resolution available if needed

Like your version of the library.  Much nicer. 

Thanks John

**Comment by barbudor** [comment]
Hi @jmfloyd 

I've added the 2 constants as:
```
# General constants
C_BUS_ADC_LSB                    = const(0.008)      # VBus ADC LSB is 8mV
C_SHUNT_ADC_LSB                  = const(0.00004)    # VShunt ADC LSB is 40µV
```
for both lite and full in this branch : https://github.com/barbudor/CircuitPython_INA3221/tree/pr_add_LSB_constants
Are you using lite or full ?

I don't have the time to setup a test platform, so could you please test and report back that I haven't broken anything ?

If this is ok, I will merge in the main branch.

Thanks for the suggestion.

**Comment by jmfloyd** [comment]
I have tested your changes and working  as expected
John

**Comment by barbudor** [comment]
Closed by https://github.com/barbudor/CircuitPython_INA3221/pull/4
Thanks

**Comment by barbudor** [comment]
Closed by #4
Thanks
